coala_quickstart/coala_quickstart.py

```diff
@@ -45,6 +45,11 @@ def _get_arg_parser():
         dest='incomplete_sections', const=True,
         help='generate coafile with only `bears` and `files` field in sections')
 
+    arg_parser.add_argument(
+        '--no-filter-by-capabilities', action='store_const',
```

Comment:
Actually, the default behavior we have in mind is to make filtering bears by capabilities default behavior. Not sure how to do this with having a positive arg